src/chains/filecoin/options/README.md

```diff
@@ -1,3 +1,36 @@
 # `@ganache/filecoin-options`
 
-> TODO: description
+This package defines the available NodeJS and CLI options for `@ganache/filecoin`
+
+## CLI Usage
+
+Run `ganache filecoin --help` to see your versions CLI usage.
+
+## NodeJS Usage
+
+See the [web documentation](#todo) for more details on the available NodeJS options.
+
+These options are provided alongside the `flavor` options. For example:
+
+```json
```

Comment:
Specifying json5 instead of json _should_ make github syntax highlighting understand comments so they aren't highlighted as errors in the diff.

```suggestion
```json5
```